frontend/fingertips-frontend/components/table/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { Table } from 'govuk-react';
+import { WeatherForecast } from '@/generated-sources/api-client';
+
+interface tableProps {
+  data: WeatherForecast[];
+}
+
+export function PlainTable({ data }: Readonly<tableProps>) {
+  return (
+    <div data-testid="plainTable-component">
+      <Table
+        head={
+          <Table.Row>
+            <Table.CellHeader date>Date</Table.CellHeader>
+            <Table.CellHeader numeric>Temperature C</Table.CellHeader>
+            <Table.CellHeader numeric>Temperature F</Table.CellHeader>
+            <Table.CellHeader>Summary</Table.CellHeader>
```

Comment:
We should be passing in these heading values as props, in the same way as we do for the `LineChart` component.